packages/cpp/src/browser/cpp-build-configurations-statusbar-element.ts

```diff
@@ -31,19 +31,18 @@ export class CppBuildConfigurationsStatusBarElement {
     protected readonly cppIdentifier = 'cpp-configurator';
 
     /**
-     * Display the cpp build configurations status bar element,
-     * and listen to any changes in the active build configuration
+     * Display the `CppBuildConfiguration` statusbar element and listen for updates.
      */
     show(): void {
         this.setCppBuildConfigElement();
-        this.cppManager.onActiveConfigChange(e => { this.setCppBuildConfigElement(); });
+        this.cppManager.onActiveConfigChange(() => this.setCppBuildConfigElement());
```

Comment:
the "onActiveConfigChange" callback receives the new active config I think?  Can we pass it to setCppBuildConfigElement, so it doesn't need to query the active config again from the manager?